testing/btest/language/ignore-deprecations.zeek

```diff
@@ -0,0 +1,71 @@
+# @TEST-EXEC: zeek -b %INPUT >out 2>&1
+# @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-remove-abspath btest-diff out
+
+module X;
+
+export {
+	global c: count = 1 &deprecated="removal planned";
+}
+event zeek_init()
+	{
+	print c;
+@pragma start-ignore-deprecations
+	print c;
+@pragma end-ignore-deprecations
+	}
+
+@TEST-START-NEXT
```

Comment:
Mixing BTest's `@` with Zeek's `@` becomes pretty hard to follow, could you put these behind comments like you already did in the test header?